src/EnergyPlus/UnitarySystem.cc

```diff
@@ -15018,24 +15012,13 @@ namespace UnitarySystems {
         }
 
         if ((ScheduleManager::GetCurrentScheduleValue(state, this->m_SysAvailSchedPtr) > 0.0) &&
-            (state.dataLoopNodes->Node(InletNode).MassFlowRate > DataHVACGlobals::SmallAirVolFlow)) {
-
-            // Determine if there is a sensible load on this system
-            if ((state.dataLoopNodes->Node(InletNode).Temp < DesOutTemp) &&
-                (std::abs(state.dataLoopNodes->Node(InletNode).Temp - DesOutTemp) > DataHVACGlobals::TempControlTol))
-                SensibleLoad = true;
+            (inletNode.MassFlowRate > DataHVACGlobals::SmallAirVolFlow)) {
 
-            if (SensibleLoad) {
+            if (inletNode.Temp < (DesOutTemp - DataHVACGlobals::TempControlTol)) {
                 if (this->m_EMSOverrideSuppCoilSpeedNumOn) {
                     this->setEMSSuppCoilStagePLR(state);
                 } else {
-                    ReqOutput = state.dataLoopNodes->Node(InletNode).MassFlowRate *
-                                Psychrometrics::PsyDeltaHSenFnTdb2W2Tdb1W1(DesOutTemp,
-                                                                           state.dataLoopNodes->Node(InletNode).HumRat,
-                                                                           state.dataLoopNodes->Node(InletNode).Temp,
-                                                                           state.dataLoopNodes->Node(InletNode).HumRat);
-                    // Get no load result
-                    PartLoadFrac = 0.0;
```

Comment:
ReqOutupt and FullOutput are only used in warning messages, moved to those locations instead of calculating each time through.